node/package.json

```diff
@@ -14,6 +14,6 @@
   },
   "overrides": {
     "jsonwebtoken": "9.0.0",
-    "axios": "1.8.2"
+    "axios": "1.12.0"
```

Comment:
As far as I can tell, these overrides were due to the compatibility-api problems, and those were recently fixed by Donald.

This repo uses the `@signalwire/node` package, which uses the `@signalwire/web-api` package and that depends on the `@signalwire/compatibility-api`. 

So, this repo is indirectly using the `@signalwire/compatibility-api` and it seems we only have released the compatibility-api package but the dependent packages are still using the old version of the compatibility-api. If we update these packages, then we might be able to remove overrides from here.